marklogic-data-hub-central/src/test/java/com/marklogic/hub/central/controllers/model/ModelControllerTest.java

```diff
@@ -60,26 +79,115 @@ private void updateModelInfo() {
     }
 
     private void updateModelEntityTypes() {
+        // Loading unrelated indexes so that we can check for them after updating entity model
```

Comment:
Food for thought for the future - no change needed now:

"NounTest" - e.g. "ModelControllerTest" - is usually a decent approach for the first couple tests. But that doesn't scale well as the "Noun" becomes able to do more "Verbs". At that points, it's worth refactoring into "Verb1Test", "Verb2Test", etc, and putting them into a package specific to the Noun. 

I think ModelControllerTest isn't too big yet, but when we add more tests, it's likely going to be worth moving to a VerbTest approach.